pkgs/top-level/python-packages.nix

```diff
@@ -16640,16 +16640,16 @@ EOF
 
   canonicaljson = buildPythonPackage rec {
     name = "canonicaljson-${version}";
-    version = "1.0.0";
+    version = "1.1.3";
 
     src = pkgs.fetchgit {
```

Comment:
`fetchFromGitHub`, or even better - if possible - `fetchPypi`

Reply:
should work with `fetchPypi`, did not know that existed, on it.